frontends/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "ipa-frontend",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "dev": "next -p 7759c",
+    "build": "next build",
+    "build:css": "postcss src/style/tailwind.css -o src/style/main.css",
+    "build:watch": "postcss src/tailwind.css -o src/main.css --watch",
```

Comment:
the `main.css` file is never consumed, so I believe you can delete this
also, this task is never run